### DISCUSSION TITLE
How Can We Click Icons in Text Inputs?

### DISCUSSION THREAD

**Discussion by jschnall** [beercss/beercss#299] [General]
For some reason it doesn't seem to be possible to add an onClick listener to < i > tags. I can wrap them with a div and add the listener to that element instead, but doing so bumps the icon out of the text field. Ideas?

**Comment by leonardorafael** [comment]
Hi, 

You can use the `front` helper (https://github.com/beercss/beercss/blob/main/docs/HELPERS.md#positions). 
Also you can wrap the icon with the `a` tag:

```html
<div class="field border prefix">
  <i class="front">search</i>
  <input>
</div>
```

```html
<div class="field border prefix">
  <a>
    <i>search</i>
  </a>
  <input>
</div>
```

**Reply by jschnall** [reply]
Wrapping with the < a > tag worked, thanks. I had tried several others without success.
